plugin/keyring_vault/tests/mtr/table_encrypt_3_directory-master.opt

```diff
@@ -0,0 +1 @@
+--testcase-timeout=60
```

Comment:
Is this a valid option for a `.opt` file?

Reply:
yes, it is used quite a lot